tests/kernel/threads/thread_apis/src/test_threads_set_priority.c

```diff
@@ -6,7 +6,7 @@
 
 #include <ztest.h>
 
-#define STACK_SIZE (640 + CONFIG_TEST_EXTRA_STACKSIZE)
+#define STACK_SIZE (512 + CONFIG_TEST_EXTRA_STACKSIZE)
```

Comment:
Hm, I am wondering if we could do some clean up here, btw.
I see that `tstack `and `tsize `are defined in `main.c`

Could you, instead, delete this re-defnition of STACK_SIZE and make the k_thread_create() below use the tstack and tsize?

If this can be done, please, do this in all tests in this suite :)